presidio-anonymizer/presidio_anonymizer/engine/anonymizers/hash.py

```diff
@@ -0,0 +1,13 @@
+import hashlib
+
+# TODO implement + test
+from anonymizers.anonymizer import AnonymizerAbstract
+
+
+class Hash(AnonymizerAbstract):
+    def __init__(self,
+                 old_text: str):
+        self.text = old_text
+
+    def annonymize(self):
+        return hashlib.sha256(self.old_text.hash_str.encode()).hexdigest()
```

Comment:
lots of magic here, maybe extract to variables for descriptiveness?

Reply:
sure.